config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml

```diff
@@ -81,8 +81,8 @@ periodics:
       - ./test/e2e/testing-manifests/auth/encrypt/run-e2e.sh
```

Comment:
TODO: something like https://github.com/kubernetes/test-infra/issues/33980 & take advantage of `kind build node-image` now supporting consuming Kubernetes release + CI builds instead of compiling kubernetes redundantly in periodics.

(it will make the jobs faster and make it possible to use say 2 cores for this job, but it requires some work)